tests/unit/tabular/test_base.py

```diff
@@ -4,9 +4,11 @@
 import pytest
 
 from sdv.metadata.table import Table
+from sdv.tabular.base import COND_IDX
 from sdv.tabular.copulagan import CopulaGAN
 from sdv.tabular.copulas import GaussianCopula
 from sdv.tabular.ctgan import CTGAN, TVAE
+from tests.utils.utils import DataFrameMatcher
```

Comment:
Why is this `utils.utils`?
I think it could be directly put under `tests/utils.py`